frontend/src/BrowseReceiving/BrowseReceiving.tsx

```diff
@@ -1,111 +1,113 @@
 import React, { FC, useEffect, useState } from 'react';
 import browseReceivingQuery, { Received } from './browseReceivingQuery';
-import { Grid, Box } from '@material-ui/core';
+import { Grid, Box, Typography, Button } from '@material-ui/core';
 import BrowseReceivingItem from './BrowseReceivingItem';
 import moment from 'moment';
-import { useFormik } from 'formik';
-import { Form } from 'semantic-ui-react';
 import Loading from '../ui/Loading';
-import FormikControlledSearchBar from '../ui/FormikControlledSearchBar';
-import { FormikNativeDatePicker } from '../ui/FormikNativeDatePicker';
 import { HeaderText, SectionText } from '../ui/Typography';
+import BrowseReceivingFilterDialog, {
+    FormValues,
+} from './BrowseReceivingFilterDialog';
+import Placeholder from '../ui/Placeholder';
 
-interface FormValues {
+interface Filters {
     cardName: string;
     startDate: string;
     endDate: string;
 }
 
-const initialFormValues: FormValues = {
+const initialFilters: Filters = {
     cardName: '',
     startDate: moment().subtract(30, 'days').format('YYYY-MM-DD'),
     endDate: moment().format('YYYY-MM-DD'),
 };
 
-// No validations needed for now
-const validate = () => {
-    return {};
-};
+function shallowCompare(obj1: Filters, obj2: Filters) {
+    return JSON.stringify(obj1) === JSON.stringify(obj2);
+}
```

Comment:
This is fragile and entirely dependent on the filter objects having the same property order. If this becomes problematic, we'll have to consider other options.